lib/components/Home/Sidebar.dart

```diff
@@ -154,11 +154,19 @@ class Sidebar extends HookConsumerWidget {
                                     backgroundImage:
                                         CachedNetworkImageProvider(avatarImg),
                                   ),
-                                  const SizedBox(width: 10),
-                                  Text(
-                                    data.displayName ?? "Guest",
-                                    style: const TextStyle(
-                                      fontWeight: FontWeight.bold,
+                                  const SizedBox(
+                                    width: 10,
+                                  ),
+                                  SizedBox(
+                                    width: 120,
```

Comment:
you can use `Flexible` instead of a fixed `SizedBox` :smiley:

Reply:
Good idea! (this is my first time ever using flutter so I apologize for these oversights)